quinn-proto/src/config.rs

```diff
@@ -782,8 +806,17 @@ impl ServerConfig {
         crypto: Arc<dyn crypto::ServerConfig>,
         token_key: Arc<dyn HandshakeTokenKey>,
     ) -> Self {
+        let transport = Arc::new(TransportConfig::default());
+        let max_buffer_bytes_per_incoming = transport.receive_window.into_inner().min(
+            transport
+                .max_concurrent_bidi_streams
+                .into_inner()
+                .saturating_add(transport.max_concurrent_uni_streams.into_inner())
+                .saturating_mul(transport.stream_receive_window.into_inner())
+                .saturating_add(1200),
+        );
```

Comment:
This is complex and prone to misconfiguration (e.g. datagram-only connection). Maybe just set it to, say, 1/10th of the default global max?

Reply:
Done.